README.md

```diff
@@ -180,6 +180,21 @@ Applications may set Marathon labels to override load balancer settings for each
 | com.meltwater.proxymatic.&lt;N&gt;.maxconn     | Maximum concurrent connections per container.                                                |
 | com.meltwater.proxymatic.&lt;N&gt;.mode        | Load balancer mode for this port as either `tcp` or `http`. Default is `tcp` mode.           |
 
+Or set Marathon labels to override load balancer settings for the service client or server timeouts. For example
+
+```
+  "labels": {
+    "com.meltwater.proxymatic.service.timeoutclient": 300,
```

Comment:
The timeouts are a setting on a specific servicePort and a service can have multiple ports. I'd suggest using the same parameter naming scheme as the other settings, like so

```
com.meltwater.proxymatic.port.<N>.timeout.client
com.meltwater.proxymatic.port.<N>.timeout.server
```